package/CHANGELOG

```diff
@@ -19,6 +19,9 @@ mm/dd/18 richardjgowers, palnabarun, bieniekmateusz, kain88-de, orbeckst,
   * 0.17.1
 
 Enhancements
+  * Can now pass multiple attributes as a list to groupby function.
```

Comment:
@davidercruz also add your GitHub hand *davidercruz* to the list of authors above (after ayshsuhane)